File: POO/EJ1PY/Autora.py
class Autora:
    # Atributos
    nombre = ""
    apellido = ""
    #Método constructor
    def Autora(self,pnombre,papeliido):
        self.nombre = pnombre
        self.apellido = papeliido

    # Getters y setters

    def getNombre(self):
        return self.nombre

    def setNombre(self,pnombre):
        self.nombre = pnombre

    def getApellido(self):
        return self.apellido

    def setApellido(self,papellido):
        self.apellido = papellido

File: POO/EJ1/EJ1JAVA/Autora.java
package POO.EJ1JAVA;

public class Autora {
    private String nombre;
    private String apellidos;

    public Autora(String pnombre, String papellidos) {
        this.nombre = pnombre;
        this.apellidos = papellidos;
    }

    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String pnombre) {
        this.nombre = pnombre;
    }

    public String getApellidos() {
        return this.apellidos;
    }

    public void setApellidos(String papellidos) {
        this.apellidos = papellidos;
    }
}
